enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,346 @@
+---
+title: automated-backups-no-config
+authors:
+  - "@elbehery"
+reviewers:
+  - "@JoelSpeed"
+  - "@dusk125"
+  - "@hasbro17"
+  - "@tjungblu"
+  - "@vrutkovs"
+  - "@jmhbnz"
+  - "@deads2k"
+  - "@csrwng"
+  - "@sjenning"
+  - "@wking"
+  - "@rhuss"
+  - "@gnufied"
+approvers:
+  - "@deads2k"
+  - "@JoelSpeed"
+  - "@hasbro17"
+  - "@tjungblu"
+  - "@vrutkovs"
+  - "@JoelSpeed"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2024-06-17
+last-updated: 2024-10-21
+tracking-link:
+  - https://issues.redhat.com/browse/ETCD-609
+see-also:
+  - "https://issues.redhat.com/browse/OCPSTRAT-1411"
+  - "https://issues.redhat.com/browse/OCPSTRAT-529"
+---
+
+
+# Automated Backups of etcd with No Config
+
+## Summary
+
+To enable automated backups of Etcd database and cluster resources of an Openshift cluster from day one after installation, 
+without provided configuration from the user.
+
+This document outlines the possible approaches that were investigated, discussed and tested, in addition to the final implemented approach.  
+
+## Motivation
+
+The current method of **Backup** and **Restore** of an Openshift cluster is outlined in  [Automated Backups of etcd](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md). 
+This approach relies on user supplied configuration, in the form of `config.openshift.io/v1alpha1 Backup` CR. This method utilizes `batchv1.CronJob` in order to take Backups of Etcd and cluster resources.
+
+To improve the user's experience, this work proposes taking etcd backup using default configurations from day one, without user intervention.
```

Comment:
question: from a support perspective does this mean it is a bug if the backup is invalid or corrupt? are there any legal concerns with Red Hat "owning" backups?